internal/apps/cmp/impl/clusters/add_edge_cluster.go

```diff
@@ -178,7 +176,8 @@ func (c *Clusters) AddClusters(ctx context.Context, req apistructs.CloudClusterR
 	}
 	logrus.Infof("add edge cluster yaml: %v", string(b))
 
-	dto, err := c.pipelineSvc.PipelineCreateV2(ctx, &pipelinepb.PipelineCreateRequestV2{
+	dto, err := c.pipelineSvc.PipelineCreateV2(transport.WithHeader(ctx,
```

Comment:
use `pkg/common/apis/context.go#WithInternalClientContext` and `pkg/discover`.

Reply:
adjusted